activesupport/lib/active_support/duration.rb

```diff
@@ -180,13 +180,15 @@ def since(time = ::Time.current)
       sum(1, time)
     end
     alias :from_now :since
+    alias :after :since
 
     # Calculates a new Time or Date that is as far in the past
     # as this Duration represents.
     def ago(time = ::Time.current)
       sum(-1, time)
     end
     alias :until :ago
+    alias :before :ago
```

Comment:
Still feels weird to alias these—and leave ago's required argument as optional—rather than define new methods. Not necessary for this PR, but worth considering

```ruby
# Calculates this duration in the past relative to +date_or_time+.
def before(date_or_time)
  sum(-1, date_or_time)
end
alias :until :before

def ago
  before ::Time.current
end
```

That'd break people using the odd `2.days.ago customer_start_date` and `5.weeks.since` formulations, though, so it'd need deprecation 😐